### PR TITLE
WPushButton debug tooltip

### PR DESCRIPTION
**Pull request by ronso0**
* restore debug tooltip
* merge Enter & Leave event into event()

### CODE REVIEW

File: src/widget/wpushbutton.cpp [hunk 1]
```diff
@@ -444,6 +432,14 @@ bool WPushButton::event(QEvent* e) {
             m_bPressed = false;
             restyleAndRepaint();
         }
+    } else if (e->type() == QEvent::ToolTip) {
+        updateTooltip();
+    } else if (e->type() == QEvent::Enter) {
+        m_bHovered = true;
```
Comment (daschuer) [inline src/widget/wpushbutton.cpp]: Is this hack still required? I had the impression that this is not the case during development of the HotcueButton.

Comment (ronso0) [reply src/widget/wpushbutton.cpp]: yes.
`[hover="true"]` allows to change the font color, default Qt `:hover` doesn't.

Comment (daschuer) [reply src/widget/wpushbutton.cpp]: It was introduced here recently:  https://github.com/mixxxdj/mixxx/pull/2471
So I think it is required ..

Comment (ronso0) [reply src/widget/wpushbutton.cpp]: yup, that PR was from me :) Just tested again: with `:hover` the font won't change.

**Comment by daschuer** [comment]
Thank you. 

LGTM .

**Pull request merged**